source/type/pb_converter.cpp

```diff
@@ -60,6 +61,48 @@ static void fill_codes(const NT* nt, const nt::Data& data, PB* pb) {
     }
 }
 
+template<typename PB>
+static void fill_object_from_ptref(PB* pb,
+                                   const navitia::type::Type_e type_e,
+                                   const nt::Data& data,
+                                   std::string& request,
+                                   int depth,
+                                   const pt::ptime& now,
+                                   const pt::time_period& action_period,
+                                   const bool show_codes) {
```

Comment:
I don't like that. We can do much better. But maybe it can be rewritten later by the pb_convertor refactor?